<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello there!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 63 oblique 11pt &quot;URW Gothic L&quot;;
 </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Users:</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRegister">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>User:
&lt;strong&gt;None&lt;/strong&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>241</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../progbase2/progbase2/labs/lab7/data/images.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>201</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Knopochki))</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Bmp Secrets</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Check password</string>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPassCheck">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassCheck">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Your password</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Service</string>
     </attribute>
     <widget class="QFrame" name="infoBlock">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>251</width>
        <height>281</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="midLineWidth">
       <number>3</number>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>231</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Manager</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Managing:</string>
        </property>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>106</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>106</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>106</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonNextPage">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>51</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPageNumber">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPrevPage">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>180</y>
        <width>51</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Templates</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetTemplates">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>10</y>
        <width>241</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidgetTemplates">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>251</width>
        <height>471</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTemplatesButtons">
       <attribute name="title">
        <string>Manager</string>
       </attribute>
       <widget class="QGroupBox" name="groupBoxTemplatesButtons">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Managing:</string>
        </property>
        <widget class="QPushButton" name="pushButtonAddTemplates">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>106</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonEditTemplates">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>106</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonDeleteTemplates">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>106</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>231</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Import/Export</string>
        </property>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonExport">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Share</string>
        </property>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>161</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Share Template</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tabTemplatesSearch">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QLineEdit" name="lineEditTemplatesSearch">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonTemplatesSearchClear">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="infoBlockTemplates">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="midLineWidth">
       <number>3</number>
      </property>
      <widget class="QLabel" name="labelTemplates_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelTemplates_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelTemplates_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Gothic L</family>
         <pointsize>11</pointsize>
         <weight>7</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowserT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>231</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>/cat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLog_in"/>
    <addaction name="actionRegister"/>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="title">
     <string>Service</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuTemplate">
    <property name="title">
     <string>Template</string>
    </property>
    <addaction name="actionadd"/>
    <addaction name="actionEdit_2"/>
    <addaction name="actionDelete_2"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUser"/>
   <addaction name="menuService"/>
   <addaction name="menuTemplate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit </string>
   </property>
  </action>
  <action name="actionLog_in">
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEdit_2">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete_2">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
